Sources/Quick/Configuration/QuickConfiguration.swift

```diff
@@ -0,0 +1,8 @@
+import XCTest
+
+// NOTE: This file is not intended to be included in the Xcode project. It
+//       is picked up by the Swift Package Manager during its build process.
```

Comment:
Great note. Similar files in swift-package-manager always confuse me; it takes a long time to grok the build system.